src/lib/helpers_test.go

```diff
@@ -30,44 +32,61 @@ func setupTestVars(chunkSize int) {
 	LOCK_TRIES = 3
 }
 
-func addRecords(cb RecordSetupCB, blockCount int) []*Record {
+// getTestTableName uses the caller as the test name to help provide test case isolation.
+func getTestTableName(t *testing.T) string {
+	t.Helper()
```

Comment:
t.Helper is a go 1.9 feature, let's keep compatibility with at least 1.6 (which i'm still on from ubuntu LTS 16.04)